From 3a87568927cf622376207b6786ae7db465e0aacf Mon Sep 17 00:00:00 2001
From: Komal Bajaj <quic_kbajaj@quicinc.com>
Date: Wed, 11 Oct 2023 12:11:35 +0530
Subject: [PATCH 05/27] QCLINUX: arm64: dts: qcom: Add board-id and msm-id for
 QCM6490

Add board-id and msm-id for QCM6490-idp for now. This is only a
workaround, that shall be replaced by the compatible string
check approach to pick the correct DTB.

Change-Id: If7b1590b9ea41d607752b59db1f49bbb0b204d7b
Signed-off-by: Komal Bajaj <quic_kbajaj@quicinc.com>
Signed-off-by: Salendarsingh Gaud <quic_sgaud@quicinc.com>
Signed-off-by: Atul Dhudase <quic_adhudase@quicinc.com>
Upstream-Status: Pending
---
 arch/arm64/boot/dts/qcom/qcm6490-idp.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
index 9b1bf7d1c98d..02643bd5b78d 100644
--- a/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
+++ b/arch/arm64/boot/dts/qcom/qcm6490-idp.dts
@@ -16,6 +16,10 @@ / {
 	model = "Qualcomm Technologies, Inc. QCM6490 IDP";
 	compatible = "qcom,qcm6490-idp", "qcom,qcm6490";
 
+	/* This will be deprecated soon */
+	qcom,msm-id = <497 0x10000>, <498 0x10000>, <475 0x10000>, <515 0x10000>;
+	qcom,board-id = <34 0>, <34 1>;
+
 	aliases {
 		serial0 = &uart5;
 	};
-- 
2.25.1

